lizard_auth_server/views_api_v2.py

```diff
@@ -180,7 +179,7 @@ def form_valid(self, form):
         """
         # The JWT message is validated; now check the message's contents.
         if (('username' not in form.cleaned_data) or
-            ('password' not in form.cleaned_data)):
```

Comment:
Deze spatie hoort er wel (er staan haakjes om de beide regels heen).

Reply:
Dat vind ik zelf eigenlijk ook maar pyflakes is het met ons beide oneens...

Heb nu een iets andere smaak gebruikt, wat vind jij daar van??